examples/reagent/src/cljs/todomvc/components/todo_edit.cljs

```diff
@@ -0,0 +1,30 @@
+(ns todomvc.components.todo-edit
+  (:require [reagent.core :as reagent]
+            [todomvc.actions :as actions]
+            [todomvc.helpers :as helpers]))
+
+(defn on-key-down [k id title default editing]
+  (let [key-pressed (.-which k)]
+    (condp = key-pressed
+      helpers/enter-key (actions/save-todo id title editing)
+      helpers/escape-key (do (reset! title default)
+                             (reset! editing false))
+      nil)))
+
+(defn component-render [{:keys [id title completed]} editing]
+  (let [default title
+        edit-title (reagent/atom default)]
+    (fn []
```

Comment:
I think the reagent community has defaulted to using (fn [ ] element) instead of #( ) in cases like this.